src/views/product-integrations-landing/components/searchable-integrations-list/index.tsx

```diff
@@ -313,13 +352,13 @@ function MobileFilters() {
 	const makeToggleTierHandler = (tier: Tier) => () => {
 		switch (tier) {
 			case Tier.OFFICIAL:
-				setOfficialChecked((s) => !s)
+				setOfficialChecked(!officialChecked)
 				break
 			case Tier.PARTNER:
-				setPartnerChecked((s) => !s)
+				setPartnerChecked(!partnerChecked)
 				break
 			case Tier.COMMUNITY:
-				setCommunityChecked((s) => !s)
+				setCommunityChecked(!communityChecked)
```

Comment:
I changed the API of these setter functions — no longer accepts a `React-setState-style` callback as an arg... same end behavior otherwise